src/app/core/webdav/cloud-file-service.service.spec.ts

```diff
@@ -151,14 +149,4 @@ describe("CloudFileService", () => {
       "image"
     );
   });
-
```

Comment:
This test got removed because the assertion did not get triggered before and the function `doesFileExist` is not even called by the function. I think this test got committed by accident.